### PR TITLE
Add support for multi language

### PR DESCRIPTION
**Pull request by dnlkoch**

### CODE REVIEW

File: src/en/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+![](./assets/geoserver-logo.png)
+
+# Introduction to GeoServer
+
+Welcome to the **Introduction of GeoServer - Workshop**.
```
Comment (dnlkoch) [inline src/en/README.md]:
```suggestion
Welcome to the **Introduction to GeoServer - Workshop**.
```

File: src/en/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+This workshop was developed for use on [OSGeo-Live {{ book.osGeoLiveVersion }} DVD](https://live.osgeo.org) and is intended to give you a comprehensive overview of GeoServer as a web mapping solution. <!---Hinweis auf DVD? -->
```
Comment (dnlkoch) [inline src/en/README.md]:
```suggestion
This workshop is developed to be used within the [OSGeo-Live {{ book.osGeoLiveVersion }}](https://live.osgeo.org) environment and is intended to give you a comprehensive overview of GeoServer as a web mapping solution.
```

Note about the DVD is not needed here.

File: src/en/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+> **info**
+>
+> The workshop can be downloaded [as PDF here]({{ book.workshopPdfUrl }}).  <!---PDF auf englisch? -->
```
Comment (dnlkoch) [inline src/en/README.md]: To fix the download links please proceed as follows: 
* Replace the variable `workshopPdfUrl` (in book.json) with `workshopPdfUrlEn` and `workshopPdfUrlDe`
* Set the respective values to `https://terrestris.github.io/geoserver-introduction-ws/geoserver-introduction-ws_en.pdf`/`https://terrestris.github.io/geoserver-introduction-ws/geoserver-introduction-ws_de.pdf`
* Adjust the variable names in the text

File: src/en/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+Publish a layer group
+
+* [Styling:](styling/README.md)
+Style a layer with SLD (Styled Layer Descriptor)
+
+# Authors
+
+{% for author in book.authors %}
+  - {{ author.name }} ([{{ author.mail }}](mailto:{{ author.mail }}))
+{% endfor %} <!--alle Autoren aktuell?-->
```
Comment (dnlkoch) [inline src/en/README.md:42]: Please add yourself to the list of authors.

File: src/en/basics/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+The *stable* in the respective URL can also be replaced by a version number, if you want to access the documentation of a specific GeoServer version. Within this workshop **Version {{ book.geoServerVersion }}** is used, so the resulting URL for the user-documentation is <https://docs.geoserver.org/{{ book.geoServerVersion }}/user/>.
+
+![GeoServer web interface after successful login.](../assets/geoserver_login_gui.png)
+
+Usually, GeoServer is provided for productive operation as a (Java-)standalone servlet in the form of a `.war` - file, which can be downloaded from <http://geoserver.org/download/>.
+The `.war` - file must then be published (often called *deploy*) to a
+servlet container (for example [Tomcat](https://tomcat.apache.org/) or
+[Jetty](https://eclipse.org/jetty/)) afterwards. The web interface of GeoServer can then be called from the browser.
+
+<!--hier auf einmal deutsches bild-->
```
Comment (dnlkoch) [inline src/en/basics/README.md]: If it's the only one (and easy to create) free to add an english version of it.

File: src/en/basics/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+>
+> GeoServer is already pre-installed on the OSGeoLive system
+> and can be accessed during the workshop at   {{ book.geoServerBaseUrl }} 
+> (see [here](../environment/README.md)). This variant differs from the classical *deployment* as a .war file, since a Java program 
+> (start.jar) is executed, which programmatically  starts a Jetty-server with
+> Geoserver. This is not important for the contents of the workshop.
+
+In [the following](../ui/index.html) section we will first get an overview of the GeoServer folder structure. Afterwards it is explained how GeoServer extensions can be installed. 
+In the conclusion of the module, it is mentioned how the source code of GeoServer (or individual extensions) can be compiled with the 
+*Build-Management-Tool* Maven.
```
Comment (dnlkoch) [inline src/en/basics/README.md]: The description of the following chapter is wrong, should be something about the UI and not maven.

File: src/en/environment/README.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+# Preparatory work and general information
+
+Before you can start with the workshop, please complete the following steps:
+
+* Boot your computer including OSGeoLive-Medium
+* Choose your language
+* Choose *try ubuntu without installation* <!--in englisch richtig?-->
```
Comment (dnlkoch) [inline src/en/environment/README.md:7]: Yes, should be ok.

File: src/en/ui/data/README.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+  (e.g. in QGIS). There it is relevant for the initial map section.
+* `Coordinate Reference System`: Coordinate Reference System as EPSG-Code.
+* `Layer`: The buttons **Add Layer** or **Add Layer Group** can be used to add individual  layers or other grouped layers to the layer group.
+  A click on the corresponding button opens the *Select Layer* window, in which all available layers of the GeoServer instance appear.
+  A layer can be selected from the window by clicking on the layer's name and is then passed to the layer group. Each selected layer then appears in the table below the layer form element.
+
+In the overview table the order of the layers can be determined by the arrow symbols, where the uppermost layer is drawn as the lowest layer in this layer group. You can also assign a different style to the layer or remove the layer from the group.
+
+#### Layer style
+<!--englische Begriffe klären-->
```
Comment (dnlkoch) [inline src/en/ui/data/README.md]: Do you have a particular question here?

File: src/en/ui/data/README.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+
+![Layer style.](../../assets/ui_styles.png)
+
+The drawing rule determines the appearance of a layer depending on attribute properties. GeoServer displays the symbology of a layer with the help of SLD
+(Styled Layer Descriptor, a XML based dialect), which can be created or changed in the styles dialog.
+
+The list contains all available styles of this GeoServer and allows you to either create a new style by clicking the *Add new Style* button,
+delete an existing style (Select the style using the checkbox and click *Delete selected styles*) or changing an existing style by clicking on the style's name.
+<!--englische begriffe-->
```
Comment (dnlkoch) [inline src/en/ui/data/README.md]: Do you have a particular question here?

File: src/en/ui/server/README.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+![Server Status](../../assets/ui_server_status.png)
+
+Under **Server Status** you can find information about the current state of the map server.
+In addition to various system specifications 
+(e.g. available fonts, Java version used or memory currently in use), this includes installation specifics such as the currently used data directory. The status page is the first port of call if the performance of a layer does not match the usual speed or changes were made at the file level of GeoServer (e.g. installation of new fonts, changes in configuration files).
+Changes at file level are not recommended, but they cannot be prevented if, for example, the configuration of another GeoServer is adopted. In this case, the configuration must be set with the button **Load new**
+under **Configuration and Catalog**. If the user interface responds
+very sluggish or layers appear only very slowly in the application it can often be helpful to clear the resource cache with the button **Clear** and/or release the working memory via **Free memory**.
+<!--englische Begriffe?-->
```
Comment (dnlkoch) [inline src/en/ui/server/README.md]: Do you have a particular question here?

**Review by dnlkoch — COMMENTED** [review]
Very nice work @fschmidt56!

I left some remarks about your comments you might want to resolve.

File: src/en/basics/README.md (hunk 6 above)
Comment (k-rmrc) [reply src/en/basics/README.md]: also wrong in German version, both versions edited

**Pull request merged**